tcms/xmlrpc/__init__.py

```diff
@@ -17,7 +20,3 @@
 
 def get_version():
     return XMLRPC_VERSION
-
-
-from tcms.xmlrpc.filters import autowrap_xmlrpc_apis
-autowrap_xmlrpc_apis(__path__, __package__)
```

Comment:
I just found this change causes following error

```
(nitrate)[cqi@cqi-laptop Nitrate]$ ./manage.py runserver --traceback
Traceback (most recent call last):
  File "/home/cqi/Virtualenvs/nitrate/lib/python2.7/site-packages/django/core/management/base.py", line 222, in run_from_argv
    self.execute(*args, **options.__dict__)
  File "/home/cqi/Virtualenvs/nitrate/lib/python2.7/site-packages/django/core/management/base.py", line 250, in execute
    translation.activate('en-us')
  File "/home/cqi/Virtualenvs/nitrate/lib/python2.7/site-packages/django/utils/translation/__init__.py", line 90, in activate
    return _trans.activate(language)
  File "/home/cqi/Virtualenvs/nitrate/lib/python2.7/site-packages/django/utils/translation/trans_real.py", line 183, in activate
    _active.value = translation(language)
  File "/home/cqi/Virtualenvs/nitrate/lib/python2.7/site-packages/django/utils/translation/trans_real.py", line 172, in translation
    default_translation = _fetch(settings.LANGUAGE_CODE)
  File "/home/cqi/Virtualenvs/nitrate/lib/python2.7/site-packages/django/utils/translation/trans_real.py", line 154, in _fetch
    app = import_module(appname)
  File "/home/cqi/Virtualenvs/nitrate/lib/python2.7/site-packages/django/utils/importlib.py", line 35, in import_module
    __import__(name)
  File "/home/cqi/code/Nitrate/tcms/xmlrpc/__init__.py", line 16, in <module>
    autowrap_xmlrpc_apis(__path__, __package__)
  File "/home/cqi/code/Nitrate/tcms/xmlrpc/filters.py", line 85, in autowrap_xmlrpc_apis
    _wrap_exceptions(module_name)
  File "/home/cqi/code/Nitrate/tcms/xmlrpc/filters.py", line 36, in _wrap_exceptions
    module = __import__(module_name, {}, {}, [""])
  File "/home/cqi/code/Nitrate/tcms/xmlrpc/api/__init__.py", line 13, in <module>
    from version import *  # NOQA
  File "/home/cqi/code/Nitrate/tcms/xmlrpc/api/version.py", line 4, in <module>
    from tcms.xmlrpc import get_version
ImportError: cannot import name get_version
```

I'll make a commit to fix this right away.